src/UnitTests/Core/Commands/ScheduleCommandTests/ProcessShould.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using DevChatter.Bot.Core.Commands;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Events.Args;
+using DevChatter.Bot.Core.Systems.Chat;
+using Moq;
+using System.Collections.Generic;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Data.Specifications;
+using Xunit;
+
+namespace UnitTests.Core.Commands.ScheduleCommandTests
+{
+    public class ScheduleCommandShould
+    {
+        private readonly ScheduleCommand _scheduleCommand;
+        private readonly Mock<IChatClient> _chatClientMock = new Mock<IChatClient>();
+        private readonly Mock<IRepository> _repositoryMock = new Mock<IRepository>();
+        private readonly CommandReceivedEventArgs _commandReceivedEventArgs = new CommandReceivedEventArgs();
+
+        public ScheduleCommandShould()
+        {
+            var entities = new List<ScheduleEntity>
+            {
+                new ScheduleEntity { ExampleDateTime = new DateTimeOffset(2018, 6, 19, 18, 0, 0, 0, TimeSpan.Zero)},
+                new ScheduleEntity { ExampleDateTime = new DateTimeOffset(2018, 6, 21, 16, 0, 0, 0, TimeSpan.Zero)},
+            };
+            _repositoryMock.Setup(x => x.List(It.IsAny<DataItemPolicy<ScheduleEntity>>())).Returns(entities);
+            _scheduleCommand = new ScheduleCommand(_repositoryMock.Object);
+        }
+
+        [Theory]
+        [InlineData("-19")]
+        [InlineData("+999")]
+        [InlineData("19")]
+        [InlineData("stoptryingtobreakstuff")]
+        public void SendErrorMessage_GivenInvalidArguments(string argument)
+        {
+            List<string> arguments = new List<string>
+            {
+                argument
+            };
+            _commandReceivedEventArgs.Arguments = arguments;
+            _scheduleCommand.Process(_chatClientMock.Object, _commandReceivedEventArgs);
+            _chatClientMock.Verify(x => x.SendMessage(ScheduleCommand.Messages.OUT_OF_RANGE));
+        }
+
+        [Theory]
+        [InlineData("",   "Our usual schedule (at UTC +0) is: Tuesdays at 6:00 PM, Thursdays at 4:00 PM")]
+        [InlineData("-4", "Our usual schedule (at UTC -4) is: Tuesdays at 2:00 PM, Thursdays at 12:00 PM")]
+        [InlineData("+8", "Our usual schedule (at UTC +8) is: Wednesdays at 2:00 AM, Fridays at 12:00 AM")]
```

Comment:
I reduced the schedule data so these expected messages can get shorter. Verifying long strings is brittle (it will be brittle this way also).